modin/pandas/__init__.py

```diff
@@ -14,7 +14,7 @@
 import pandas
 import warnings
 
-__pandas_version__ = "2.0.2"
+__pandas_version__ = "2.0.3"
```

Comment:
Maybe we should change the warning below? Now we are supposed to support a range of pandas versions, which looks inconsistently with the warning.

Reply:
Makes sense, will change

Reply:
maybe we should remove the warning completely? or raise it only in case the 'major' pandas version is different from what we support